src/dash-playlist-loader.js

```diff
@@ -626,7 +630,8 @@ export default class DashPlaylistLoader extends EventTarget {
     // would be to update the manifest at the same rate that the media playlists
     // are "refreshed", i.e. every targetDuration.
     if (this.master && this.master.minimumUpdatePeriod) {
-      window.setTimeout(() => {
+      window.clearTimeout(this.minimumUpdatePeriodTimeout_);
```

Comment:
I think we don't intend for this to happen. If you had a timeout set to go off at 3000ms and are now(at 2500ms) trying to set one to go off at 6000ms, you should be able to. Following what `mediaUpdateTimeout` does may be what we want, where the timeout is cleared if we reload the loader or pause it